src/features/registry/registry.ts

```diff
@@ -591,6 +592,17 @@ export const token: RequestHandler = async (req, res) => {
 					authorizeRequest(req, scopes, tx),
 				]),
 		);
+
+		access.forEach((ac) =>
+			incrementRegistryImagePulls(1, {
```

Comment:
I would name the metric differently to show that it is images that have had a token validated for access rather than image pulls (you can use the image to pull 0 or 2+ times as well so it's not 1-1)